dev/build_release.py

```diff
@@ -371,10 +371,16 @@ def __gcb_build(cls, name, gradle_root, gcb_service_account, gcb_project):
     if os.path.isdir(gradle_cache):
       # Tell rmtree to delete the directory even if it's non-empty.
       shutil.rmtree(gradle_cache)
+    tar_filename = name + '-{:%Y%m%d%H%M%S}'.format(datetime.datetime.utcnow())
```

Comment:
ok, but could you feature flag this? You can default the feature to true and not provide a CLI option for it. That will make it easier to remove when this is no longer needed.

Also bucket name looks like s/gbc/gcb and would need to be parameterized to accomodate non-official builds.

Reply:
Done.